strapi/strapi
# Fix unable to rename file while uploading

Signed-off-by: harimkims <harimkims@gmail.com>



### What does it do?

Fix unable to rename file while uploading by updating name of File object

### Why is it needed?

To rename while uploading

### How to test it?

1. Create collection with Image
2. Go to /admin/content-manager/singleType
3. Drag'n drop Image to Image component
4. Click od ✏️ (Edit image)
5. Rename file to something else
6. Click on Finish
7. Click on Upload 1 asset to library
8. See if filename is changed

### Related issue(s)/PR(s)

Closes #11772
